Chapter2/Zad9.py

```diff
@@ -0,0 +1,35 @@
+'''
+Created on 13 lut 2018
+
+@author: agata.wiewiora
+
+H hours, M minutes and S seconds are passed since the midnight 
+Determine the angle (in degrees) of the hour hand on the clock face right now.
+
+'''
+
+
+H = int(input('Provide hours:'))
+M = int(input('Provide minutes:'))
+S = int(input('Provide seconds:'))
+
+if S > 60:
```

Comment:
nie musisz tego sprawdzać, jeżeli S <60 to S/60 == 0, a S % 60 == S, więc wynik będzie dokładnie ten sam